pkg/kubernetes/client/v1alpha1/client.go

```diff
@@ -28,199 +28,252 @@ import (
 )
 
 const (
-	// K8sMasterIPEnvironmentKey is the environment variable key used to
-	// determine the kubernetes master IP address
+	// K8sMasterIPEnvironmentKey is the environment variable
+	// key used to provide kubernetes master IP address
 	K8sMasterIPEnvironmentKey env.ENVKey = "OPENEBS_IO_K8S_MASTER"
 
-	// KubeConfigEnvironmentKey is the environment variable key used to
-	// determine the kubernetes config
+	// KubeConfigEnvironmentKey is the environment variable
+	// key used to provide kubernetes config
```

Comment:
is this the path for `KUBEONFIG` ? What is the value of the key here?

Reply:
I guess it is kubeconfig path ... Can we specify that ?

Reply:
I will do something to reduce this confusion. Thanks.

Reply:
done